src/main/java/eu/stamp_project/mutationtest/descartes/interceptors/stopmethods/StopMethodMatchers.java

```diff
@@ -45,9 +45,13 @@ static StopMethodMatcher isEmptyVoid() {
         return forBody(QueryStart.match(opCode(RETURN)));
     }
 
-    static StopMethodMatcher isSynthetic() {
-        return (classTree, methodTree) -> methodTree.isSynthetic();
-    }
+	static StopMethodMatcher isSynthetic() {
+		return (classTree, methodTree) -> methodTree.isSynthetic() && !methodTree.rawNode().name.startsWith("lambda$");
+	}
+
+	static StopMethodMatcher isLambda() {
```

Comment:
I think there is no need to create this `StopMethodMatcher` just the exclusion of lambdas in the `isSynthetic` matcher would be OK. Simple lambdas will be filtered by other matchers.

Reply:
ok. Done. Lambdas are now mutated by default (no more stopper).